x/auth/account.go

```diff
@@ -180,18 +180,20 @@ type BaseVestingAccount struct {
 
 // String implements fmt.Stringer
 func (bva BaseVestingAccount) String() string {
```

Comment:
In a scenario where we include a `BaseVestingAccount` into a log message, this would end up making the message multi-line. How about adopting a single-line format for the type's information? e.g. 
`Address=%s Coins=%s AccountNumber=%s ....`

Reply:
Maybe `%q` instead of `%s`, to facilitate parsing

Reply:
These are all related to the changes @jackzampolin made.

Reply:
I'm indifferent here though.

Reply:
Yeah, nonetheless they're easy to change. Really I'm not a huge fan of multiline logs